p5js/FeatureExtractor/FeatureExtractor_Image_Classification/sketch.js

```diff
@@ -33,6 +33,7 @@ function setup() {
 // A function to be called when the model has been loaded
 function modelReady() {
   select('#modelStatus').html('Base Model (MobileNet) loaded!');
+  classifier.load('./model/model.json', () => console.log('loaded json'));
```

Comment:
Probably should be consistent and use anonymous `function()` instead of `=>`? (Maybe someday we do a big pass over everything and use arrow with added tutorials?)

Reply:
fixed